toolbox/Dockerfile

```diff
@@ -15,7 +15,6 @@
 FROM ubuntu:xenial
 MAINTAINER Jared Watts <jared.watts@quantum.com>
 
-
 RUN apt-get update && \
```

Comment:
that is useful to know, thanks @bzub.  is there `ceph` tool functionality that is must have right now from kraken? or is the default of `version 10.2.5 (Jewel)` that's in the main `ubuntu:xenial` repo OK?  it has been with all my testing so far, but curious if there is a critical need for kraken yet.  thanks!